assets/css/modal-filter.css

```diff
@@ -70,32 +101,123 @@
   position: fixed;
   bottom: 0;
   left: 0;
-  width: 100%;
   background: linear-gradient(180deg, rgba(255, 255, 255, 0.00) 0%, #FFF 100%);
+  width: 100%;
 }
 
 .modal-filter--bottom-wrapper {
+  display: flex;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  justify-content: right;
+  align-items: center;
   border-top: 1px solid var(--neutral-200);
   background: var(--neutral-50);
   width: 100%;
   height: 72px;
-  display: flex;
-  align-items: center;
-  justify-content: right;
-  position: absolute;
-  left: 0;
-  bottom: 0;
 }
 
 .modal-filter--clear-btn {
-  background-color: transparent;
+  margin-right: 16px;
   border: none;
-  text-decoration: underline;
-  color: var(--primary-500);
+  background-color: transparent;
   padding: 0;
-  margin-right: 16px;
+  color: var(--primary-500);
+  text-decoration: underline;
+  font-size: var(--body-2-size);
```

Comment:
Do we need to set the font-size directly instead of using the preset classes?
https://github.com/whamester/WMDD-4885-Project/blob/ceafdb8ca87463fa3061aaff20caa38b2929d52b/assets/css/_typography.css